### PR TITLE
Feature/file name linting

### PR DESCRIPTION
**Pull request by FabianMeul**
## What
* feat: Add Woke and Unicorn packages
* feat(Import): Don't allow default exports
* chore: Update dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,37 +42,39 @@
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/github": "^8.0.2",
     "@semantic-release/npm": "^8.0.3",
-    "@types/jest": "^27.0.2",
-    "@types/react": "^17.0.34",
-    "dotenv-cli": "^4.1.0",
-    "jest": "^27.3.1",
-    "jest-extended": "^1.1.0",
+    "@types/jest": "27.0.3",
```
Comment (JaccoGoris) [inline package.json:45]: all carrets in the changed dependencies are removed, but not in the others, on purpose?

File: src/rules/import.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 module.exports = {
   rules: {
+    'import/default': 'off',
+    'import/no-default-export': [
+      'error',
```
Comment (JaccoGoris) [inline src/rules/import.js:5]: I would put this at warn

**Review by TijsM — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (FabianMeul) [reply package.json:45]: Good catch!

**Pull request merged**